index.html

```diff
@@ -10,951 +12,1067 @@
-      <div class="is-5 title">Fifth level</div>
-      <p>Aliquam sagittis rhoncus vulputate. Cras non luctus sem, sed tincidunt ligula. Vestibulum at nunc elit. Praesent aliquet ligula mi, in luctus elit volutpat porta. Phasellus molestie diam vel nisi sodales, a eleifend augue laoreet. Sed nec eleifend justo. Nam et sollicitudin odio.</p>
-      <figure>
-        <img src="./img/photo-1572958731731-2b7fdb0a9818.jpg">
-        <img src="./img/photo-1574281852611-b2f04c86a520.jpg">
-        <img src="./img/photo-1575023615386-1554d895d09d.jpg">
-        <figcaption>
-          Figure 1: Some beautiful placeholders
-        </figcaption>
-      </figure>
-      <div class="is-6 title">Sixth level</div>
-      <p>Cras in nibh lacinia, venenatis nisi et, auctor urna. Donec pulvinar lacus sed diam dignissim, ut eleifend eros accumsan. Phasellus non tortor eros. Ut sed rutrum lacus. Etiam purus nunc, scelerisque quis enim vitae, malesuada ultrices turpis. Nunc vitae maximus purus, nec consectetur dui. Suspendisse euismod, elit vel rutrum commodo, ipsum tortor maximus dui, sed varius sapien odio vitae est. Etiam at cursus metus.</p>
-    </div>
-  </div>
+		<section class="hero has-background-grey-light" aria-label="Three key features">
+			<div class="container features">
+				<div class="section-heading has-text-centered">
+					<h2 class="title is-2">Three key features</h2>
+					<h3 class="subtitle is-5">We don't hide what we do</h3>
+				</div>
 
-  <div class="hero has-background-grey-light">
-    <div class="container features">
-      <div class="section-heading has-text-centered">
-        <div class="title is-2">Three key features</div>
-        <div class="subtitle is-5">We don't hide what we do</div>
-      </div>
+				<div class="columns">
+					<div class="column is-4">
+						<div class="card is-shady">
+							<div class="card-image has-text-centered">
+								<i aria-hidden="true" class="fa fa-paw" aria-hidden="true"></i>
+							</div>
+							<div class="card-content">
+								<div class="content">
+									<div>Tristique senectus et netus et.</div>
+									<p>Purus semper eget duis at tellus at urna condimentum mattis. Non blandit massa enim nec. Integer enim neque volutpat ac tincidunt vitae semper quis. Accumsan tortor posuere ac ut consequat semper viverra nam.</p>
+									<p><a tabindex="0" role="link" aria-label="first key feature">Learn more</a></p>
```

Comment:
the tag **<a** still has the `role="link"` by default

Reply:
NVDA will still not see it in "insert+F7" mode. In the list of all links will be only links with HREF attribute.